server/test/run

```diff
@@ -13,9 +13,19 @@ unset _PBENCH_SERVER_CONFIG
 export LANG=C
 export LC_ALL=C
 
-export _tdir=$(dirname $(readlink -f ${0}))
-export _gitdir=$(dirname $(dirname ${_tdir}))
-export _tlib=${_tdir}/../lib
+export _server=$(dirname $(dirname $(realpath ${0})))
+export _bdir=${_server}/bin
+test -d "${_bdir}" || exit 1
+export _ldir=${_server}/lib
+test -d "${_ldir}" || exit 1
+export _tdir=${_server}/test
+test -d "${_tdir}" || exit 1
+_webs=$(realpath -e ${_server}/../web-server)
+if [[ ! -d "${_webs}" ]]; then
+    echo "ERROR: failed to resolve legacy web-server directory, \"${_webs}\"" >&2
+    exit 1
+fi
+export _webs
```

Comment:
Why do you defer `export`'ing `_webs` when you didn't defer exporting any of the preceding variables?

And, if this script exits suddenly with a `1`, are we going to wish that the `_?dir` variables had had nice error messages like `_webs` does?

Reply:
> Why do you defer `export`'ing `_webs` when you didn't defer exporting any of the preceding variables?

No good reason, just missed it when I changed from a direct test to using `realpath`.  Fixed.

> And, if this script exits suddenly with a `1`, are we going to wish that the `_?dir` variables had had nice error messages like `_webs` does?

Yes, I had that change in another PR, so I've added here instead.